include/openssl/core_dispatch.h

```diff
@@ -18,6 +18,9 @@
 extern "C" {
 # endif
 
+# define OSSL_DISPATCH_END \
+    { 0, NULL }
+
```

Comment:
Please move this to `include/openssl/core.h`, located together with the definition of `OSSL_DISPATCH`

Reply:
You mean with `struct ossl_dispatch_st`?

Reply:
Sorry for the confusion.  Yes, that's what I mean.

Reply:
Ok cool.